src/store/wallet.js

```diff
@@ -46,6 +48,7 @@ export default {
     networkStatus: 'error',
     radRequestResult: null,
     transactions: [],
+    disclaimers: {},
```

Comment:
We don't need to store the raw disclaimers.They can be a constant and be used only in signData